sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -164,7 +160,7 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
             #region Snippet:DigitalTwinsSampleGetComponent
 
-            await client.GetComponentAsync(basicDtId, SamplesConstants.ComponentName);
+            await client.GetComponentAsync<Component1>(basicDtId, SamplesConstants.ComponentName);
```

Comment:
What is `Component1`?

Reply:
I ask because I think it'll be confusing in the context of a snippet. We may want to give it a name more like `MyCustomComponent`.